src/frontend/src/styles/main.css

```diff
@@ -975,11 +981,21 @@ a:hover,
  */
 
 .c-landingPage {
+  height: 100dvh;
```

Comment:
min-height? what happens if the content is higher then c-landingPage?

Reply:
I don't want it taller than the screen.

The page has two panes. The left one should match the height of the screen, while the right one is allowed to have scroll if the content doesn't fit.

This is with `height`:

https://github.com/dfinity/internet-identity/assets/4550653/577c4732-197b-43f7-892c-5b2c22d01863

This is with `min-height`:


https://github.com/dfinity/internet-identity/assets/4550653/5b5c5b01-1c35-4bed-8f14-20c88b184ae8

Does this make sense?

Reply:
aha I did not get the the other pane was scrollable on its own! Thanks for clearing that up